marklogic-data-hub-central/ui-custom/src/views/Detail.tsx

```diff
@@ -98,8 +117,8 @@ const Detail: React.FC<Props> = (props) => {
             <button onClick={handleExpandClick}>
               <span className="label">Expand All</span>
```

Comment:
Let's have the button label read "Collapse All" when clicking it will collapse all sections. It should read "Expand All" when clicking it will expand all sections. 

The button width SHOULD NOT change when the label toggles between the two label types.